src/error.rs

```diff
@@ -120,6 +122,23 @@ pub enum DeploymentError {
     },
 }
 
+#[derive(Debug, ThisError)]
+pub enum ApplicationInstanceError {
+    #[error("There is no k8s configuration associated with your application.")]
+    NamespaceNotAvailable {
+        namespace: String,
+        application: String,
+    },
+    #[error("The \"{version}\" version has no associated k8s cluster configuration.")]
+    VersionNotAvailable {
+        namespace: String,
+        version: String,
+        application: String,
+    },
+    #[error("\"{application}\" is not available  in k8s.")]
```

Comment:
```suggestion
    #[error("This application is not available  in k8s.")]
```

Reply:
I think we should show the application name, either here or in the suggestion, so the user can report the issue easily by just show this message, then the PIC will know which application is not available.